packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/AttachedFile.java

```diff
@@ -0,0 +1,65 @@
+package cloud.pangeacyber.pangea;
+
+import cloud.pangeacyber.pangea.exceptions.PangeaException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class AttachedFile {
+
+	String filename;
+	String contentType;
+	byte[] fileContent;
+
+	public AttachedFile(String filename, String contentType, byte[] fileContent) {
+		this.filename = filename;
+		this.contentType = contentType;
+		this.fileContent = fileContent;
+	}
+
+	public String getFilename() {
+		return filename;
+	}
+
+	public String getContentType() {
+		return contentType;
+	}
+
+	public byte[] getFileContent() {
+		return fileContent;
+	}
+
+	public void save(String folder, String name) throws PangeaException {
+		if (folder == null) {
+			folder = "./";
+		}
+		Path path = Paths.get(folder);
+		if (!Files.exists(path)) {
+			try {
+				Files.createDirectories(path);
+			} catch (IOException e) {
+				throw new PangeaException("Failed to create file path", e);
+			}
+			System.out.println("Directory created: " + path);
+		}
+
+		if (name == null) {
+			name = filename != null ? filename : "defaultFilename";
+		}
+
+		this.save(folder + name);
```

Comment:
consider:
```suggestion
		this.save(Paths.get(folder, name).toString());
```

Reply:
As a bonus, could drop the `toString()` in my suggestion and have `save()` accept a `Path` argument instead of `String`, then we don't need the conversion in `save()`. Or have `save()` overloads to support both argument types if that would be useful.

Reply:
Fixed. It take a Path now.